lib/api.js

```
@@ -110,6 +124,10 @@ function Api (plugins, defaultConfig) {
       }
 
       if (plug.name && typeof plug.name === 'string') {
+        if (plug.name === 'global') {
+          console.error('plugin named "global" is reserved, skipping')
```

:fire: I would throw.. this silently fails if you're not watching